crates/db_views/src/post_view.rs

```diff
@@ -60,7 +61,7 @@ use tracing::debug;
 
 fn queries<'a>() -> Queries<
   impl ReadFn<'a, PostView, (PostId, Option<PersonId>, bool)>,
-  impl ListFn<'a, PostView, PostQuery<'a>>,
+  impl ListFn<'a, PostView, (PostQuery<'a>, &'a Site)>,
```

Comment:
Rather than passing in a site to every `.list()` function. Pass in the requesting user's Site as one of the `Query` params. That seems much cleaner.